WolvenKit/Views/Tools/AssetBrowserView.xaml.cs

```diff
@@ -269,9 +273,12 @@ private void RightFileView_OnSelectionChanged(object sender, GridSelectionChange
                 }
             }
 
-            var propertiesViewModel = Locator.Current.GetService<PropertiesViewModel>();
+            if (vm.RightSelectedItems.Count == 1)
+            {
+                var propertiesViewModel = Locator.Current.GetService<PropertiesViewModel>();
 
-            propertiesViewModel.ExecuteSelectFile(vm.RightSelectedItem);
+                propertiesViewModel.ExecuteSelectFile(vm.RightSelectedItem);
```

Comment:
If we have multiple selected items, should the property view maybe show the last one from the list instead..?

Reply:
Done